airsonic-main/src/main/java/org/airsonic/player/controller/InternalHelpController.java

```diff
@@ -257,14 +258,18 @@ private void gatherScanInfo(Map<String, Object> map) {
     private void gatherIndexInfo(Map<String, Object> map) {
         SortedMap<String, IndexStatistics> indexStats = new TreeMap<>();
         for (IndexType indexType : IndexType.values()) {
-            try (IndexReader reader = indexManager.getSearcher(indexType).getIndexReader()) {
```

Comment:
IndexReader is closable, but is indirectly managed by Searcher.
Searcher resource management uses release functions.

Reply:
Shouldn't we keep it under the try-with-resources block anyway if it's closeable?